code/modules/spells/targeted/absorb.dm

```diff
@@ -0,0 +1,61 @@
+/spell/targeted/absorb
+	name = "Absorb"
+	desc = "This spell allows you to absorb the magical abilities from fallen wizards. You must be standing over their lifeless or mutilated body in order for the spell to work."
+	abbreviation = "DG"
+	user_type = USER_TYPE_SPELLBOOK
+
+	school = "evocation"
+	charge_max = 100
+	spell_flags = IS_HARMFUL
+	invocation_type = SpI_SHOUT
+	range = 0
+	cooldown_min = 200
+
+	hud_state = "wiz_disint_old"
+
+
+/spell/targeted/absorb/invocation()
+	invocation = pick("B'TES TH' DUST", "H'STA LA V'STA", "S'AY'NARA", "G'ME 'VER")
+	..()
+
+/spell/targeted/absorb/cast(var/list/targets)
+	..()
+	var/mob/living/L = holder
+	for(var/mob/living/target in targets)
+		if(ishuman(target) || ismonkey(target))
+			var/mob/living/carbon/C = target
+			if(!C.mind)
```

Comment:
I think it's possible for a mindless mob to have spells because nowadays we store spells in both the balls and the mind. You don't necessarily have to address this but it's something you might want to keep in mind.